.travis.yml

```diff
@@ -0,0 +1,25 @@
+sudo: required
```

Comment:
I was just resurrecting the old config to see if this would work at all.  As we've seen, the answer looks like it's "no".